flask-redis-centos7-atomicapp/artifacts/kubernetes/03-pod-flask.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: flask
+  labels:
+    name: flask
+    app: flask
+spec:
+  containers:
+  - name: flask
+    image: dharmit/flask
+    ports:
+    - containerPort: 5000
```

Comment:
use `targetPort` field here and reference accordingly in flask service manifest.